hugo/content/6-api-docs/index.md

```diff
@@ -0,0 +1,9 @@
++++
+chapter = true
+icon = "<b>6. </b>"
+title = "API Docs"
+weight = 0
+
++++
+
+<html><head><meta http-equiv="refresh" content="0; url=/api/html"/></head></html>
```

Comment:
Can we just add a link to `/api/html/` instead of a link to API Docs then a redirect?